tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,34 @@
+# How to use configuration files
+
+## Structure
+
+Configuration file declares validation process. Every model has to have entry in `models` list. Each entry has to contain distinct `name`, `launchers` and `datasets` sections.
+
+Example:
+
+```yaml
+models:
+  - name: model_name
+
+    launchers:
+      - framework: dlsdk
+        adapter: adapter_name
+
+    datasets:
+      - name: dataset_name
+```
+
+## Location
+
+Configuration file `accuracy-check.yml` is located in the `models` folder with model to which it belongs.
+
+`model_name.yml` file, which is located in current `configs` folder, is a link to `accuracy-check.yml` for `model_name` model.
+
+## Options
+
+1. To run configuration specify the path to the required configuration file to `-c, --config` command line.
+2. Configuration files don't contain paths to used models and weights. The model and weights are searched automatically by name of model in path specified in `-m, --models` command line option.
```

Comment:
need info about precision directory in models downloading structure. or/and example how to run some model